src/main/java/uk/gov/hmcts/reform/blobrouter/reconciliation/task/ReconciliationMailTask.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.blobrouter.reconciliation.task;
+
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import org.slf4j.Logger;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.blobrouter.config.TargetStorageAccount;
+import uk.gov.hmcts.reform.blobrouter.reconciliation.service.ReconciliationMailService;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static java.time.ZoneOffset.UTC;
+import static org.slf4j.LoggerFactory.getLogger;
+import static uk.gov.hmcts.reform.blobrouter.config.TargetStorageAccount.CFT;
+import static uk.gov.hmcts.reform.blobrouter.util.TimeZones.EUROPE_LONDON;
+
+@Component
+@ConditionalOnProperty(value = "scheduling.task.send-reconciliation-report-mail.enabled")
+public class ReconciliationMailTask {
+
+    private static final String TASK_NAME = "send-reconciliation-report-mail";
+    private static final Logger logger = getLogger(ReconciliationMailTask.class);
+
+    private static final List<TargetStorageAccount> AVAILABLE_ACCOUNTS =
+        List.of(CFT, TargetStorageAccount.CRIME);
+
+    private final ReconciliationMailService reconciliationMailService;
+
+    public ReconciliationMailTask(ReconciliationMailService reconciliationMailService) {
+        this.reconciliationMailService = reconciliationMailService;
+    }
+
+    @Scheduled(cron = "${scheduling.task.send-reconciliation-report-mail.cron}", zone = EUROPE_LONDON)
+    @SchedulerLock(name = TASK_NAME)
+    public void run() {
+        logger.info("Started {} job", TASK_NAME);
+
+        reconciliationMailService.process(LocalDate.now(UTC), AVAILABLE_ACCOUNTS);
```

Comment:
things would be a bit simpler if `reconciliationMailService` took single account as parameter and iteration would happen here, but I guess it's a bit late for that ;)

Reply:
did not want to query same statement for each account.